website/i18n/ja/docusaurus-plugin-content-docs/current/Configuration.md

```diff
@@ -289,6 +289,19 @@ Whether or not retried spec files should be retried immediately or deferred to t
 
 Type: `Boolean`<br /> Default: `true`
 
+### groupLogsByTestSpec
+
+Choose the log output view.
+
+If set to `false` logs from different Test Specs will be printed in real-time. Please note that this may result in the mixing of log outputs from different Test Specs when running in parallel.
```

Comment:
```suggestion
If set to `false` logs from different test files will be printed in real-time. Please note that this may result in the mixing of log outputs from different files when running in parallel.
```

Reply:
Done